index.js

```diff
@@ -21,6 +20,9 @@ const isPlainObj = require('is-plain-obj');
 const PCancelable = require('p-cancelable');
 const pkg = require('./package');
 
+const getMethodRedirectCodes = [300, 301, 302, 303, 304, 305, 307, 308];
+const allMethodRedirectCodes = [300, 303, 307, 308];
```

Comment:
This would be a good use for `Set()`. You could then use `.has()` instead of `.includes()` on line 50.